src/System.Xml.XPath.XDocument/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      net463-Windows_NT;
+      <!-- netfx-Windows_NT; clashes with NETStandard.Library -->
```

Comment:
I enabled most.  Some were no longer the "home" assemblies for these types.  I think we'll need to do an API compat run to decide which types need to be added and in which assembly (eg: ns1.6 home or ns2.0 home).